### PR TITLE
chore: Bump react-native-test-app to 0.3.9

### PR DESCRIPTION
**Pull request by tido64**
## Summary

Bumps `react-native-test-app` to [0.3.9](https://github.com/microsoft/react-native-test-app/compare/0.2.13...0.3.9).

## Test Plan

CI should pass.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -12662,14 +12668,14 @@ react-native-safe-area-context@~3.0.7:
   resolved "https://registry.yarnpkg.com/react-native-safe-area-context/-/react-native-safe-area-context-3.0.7.tgz#0f53de7a30d626d82936000f3f6db374ecc4b800"
   integrity sha512-dqhRTlIFe5+P1yxitj0C9XVUxLqOmjomeqzUSSY8sNOWVjtIhEY/fl4ZKYpAVnktd8dt3zl13XmJTmRmy3d0uA==
 
-react-native-test-app@^0.2.0:
-  version "0.2.13"
-  resolved "https://registry.yarnpkg.com/react-native-test-app/-/react-native-test-app-0.2.13.tgz#f8c54c1c83ce9a86139faae79cf440ff60af341e"
-  integrity sha512-gEj3Vh+Gpad/3doWXidIBQEdEIFu7dh2b6wIZ1/CkPnwmnmSkw0pCIuTBOJMfavMfwT0de1mLgXlQXBf7VyeDA==
+react-native-test-app@^0.3.0:
+  version "0.3.8"
```
Comment (tido64) [inline yarn.lock:12672]: Bumping to latest `react-native-test-app` revealed a build break when on `react-native` 0.62. The fixes for those are pending review:

- https://github.com/microsoft/react-native-test-app/pull/269
- https://github.com/microsoft/react-native-test-app/pull/270

I'll update this PR once they've been published.

**Review by Krizzu — APPROVED** [review]
Looks good 👍

**Comment by tido64** [comment]
Thanks for fixing the CI cache issue ❤️

**Pull request merged**